test/oli_web/controllers/api/payment_controller_test.exs

```diff
@@ -148,6 +149,26 @@ defmodule OliWeb.PaymentControllerTest do
 
       reset_test_payment_config()
     end
+
+    test "download .txt file with the last payment codes created", %{conn: conn} do
+      product =
+        insert(:section, %{
+          type: :blueprint
+        })
+
+      payment1 = insert(:payment, section: product, code: 123_456_789)
+      payment2 = insert(:payment, section: product, code: 987_654_321)
+      code1 = Payment.to_human_readable(payment1.code)
```

Comment:
NIT: What about using our loved pipe operator here to not define so many variables? 😄 😄